kolibri/core/device/api.py

```diff
@@ -97,6 +97,14 @@ def create(self, request, *args, **kwargs):
         if APP_KEY_COOKIE_NAME in request.COOKIES:
             app_key = request.COOKIES[APP_KEY_COOKIE_NAME]
             response_data["app_key"] = app_key
+
+        # Restart zeroconf before moving along
+        import logging
```

Comment:
Can we move the logging import and logger initialization into the module scope? And maybe we conditionalize this on whether we are provisioning the device as a subset of users device?